test/build/50_dockerfile.bats

```diff
@@ -160,6 +160,7 @@ test 7 b
 grown in 16 instructions: tmpimg
 build slow? consider enabling the new build cache
 hint: https://hpc.github.io/charliecloud/command-usage.html#build-cache
+warning: not yet supported, ignored: issue #777: .dockerignore file
```

Comment:
I wonder if it's useful to give a line saying how many warnings are being re-printed?